### PR TITLE
adds helm templates

### PR DESCRIPTION
**Pull request by xh3b4sd**
Towards https://github.com/giantswarm/giantswarm/issues/2737.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.4
```
Comment (rossf7) [inline Dockerfile]: 3.7 is the latest release.

**Review by rossf7 — APPROVED** [review]
Nitpick on the alpine version then LGTM.

File: Dockerfile (hunk 1 above)
Comment (xh3b4sd) [reply Dockerfile]: Ack. I will change this. We should then also align other projects.

Comment (rossf7) [reply Dockerfile]: I use this whenever creating new images. But you're right we should update existing images too.

**Pull request merged**